tests/code/test_transcripts.py

```diff
@@ -1,63 +1,197 @@
 import pytest
 from peewee import DoesNotExist
 
-from app.logic.transcript import getSlCourseTranscript, getProgramTranscript
-from app.models.term import Term
-from app.models.course import Course
-from app.models.courseParticipant import CourseParticipant
+from app.logic.transcript import *
 from app.models.user import User
-from app.models.eventParticipant import EventParticipant
+from app.models.courseParticipant import CourseParticipant
+from app.models.event import Event
+from app.models.programEvent import ProgramEvent
+from app.models.facilitator import Facilitator
+from app.logic.volunteers import addVolunteerToEvent
+from app.logic.events import deleteEvent
+
 
 @pytest.mark.integration
-def test_noGetSlTranscript():
+def testingTrainings():
+    user = User.create(
+    username = "namet",
+    bnumber = "B001234567",
+    email = "namet@berea.edu",
+    phoneNumber = "555-123-1234",
+    firstName = "Test",
+    lastName  = "Name",
+    isStudent = 1,
+    isFaculty = 0,
+    isCeltsAdmin = 0,
+    isCeltsStudentStaff = 0,
+    )
+
+    newEvent = Event.create(name = "Test Training Event",
+                              term = 1,
+                              description= "Event for testing",
+                              timeStart = "18:00:00",
+                              timeEnd = "21:00:00",
+                              location = "The testing lab",
+                              isRecurring = 0,
+                              isRsvpRequired = 0,
+                              isPrerequisiteForProgram = 0,
+                              isTraining = 1,
+                              isService = 0,
+                              startDate =  2021-12-12,
+                              endDate =  2021-12-13)
+
+    programEvent = ProgramEvent.create(program=2, event=newEvent)
+
+    facilitatorEntry = Facilitator.create(user = 'ramsayb2',event = newEvent)
+
+    testingTrainingEvent = Event.get(Event.name == "Test Training Event")
+
+    addVolunteerToEvent('namet', testingTrainingEvent.id, 2)
+
+    username = "namet"
+    adminName = "ramsayb2"
+
+    testingTrainingsExist = getTrainingTranscript(username)
+    testingTrainingNotExist = getTrainingTranscript(adminName)
+
+
+    assert testingTrainingNotExist.exists() == False
+    assert [event.event.name == "Test Training Event" for event in testingTrainingsExist]
+    assert testingTrainingsExist.exists()
```

Comment:
Testing if there is anything in an array should usually come before testing the contents of that array. You want to do the broader tests first, so that the specific ones don't fail for reasons other than what you are testing. It's easier to follow the test output, and if the broader ones are later they are useless, because the test will always have failed before you get to them.